js/src/panes/pane.map.js

```diff
@@ -256,18 +256,19 @@ Wu.MapPane = Wu.Pane.extend({
     },
 
     _onMove : function () {
```

Comment:
Coz when we move and /or Zoom , Project Update Functions should not be called , right ? Project Update function updates Project Name and Slug . Then why in every Mouse Move or Zoom , we need to do this heavy Server Call as we are not actually wanting to update Name and Slug

And the Event was not getting fired before at all , only the Code was there , but as the event had issue , it was doing nothing!